awestruct/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
 RUN npm install -g yarn
 
 # Install Antora framework
-RUN yarn global add @antora/cli@latest @antora/site-generator-default@latest \
+RUN yarn global add @antora/cli@2.3.0-alpha.2 @antora/site-generator-default@2.3.0-alpha.2 \
```

Comment:
yeah, better not to depend on a volatile tag such as latest anyways.

Reply:
Yep, we were lucky that the upgrade that happened transparently from 2.1 to 2.2 had no impact.